src/routes/github/create-branch/github-create-branch-post.ts

```diff
@@ -52,12 +93,18 @@ export const GithubCreateBranchPost = async (req: Request, res: Response): Promi
 		statsd.increment(metricCreateBranch.created, tags);
 	} catch (err) {
 		req.log.error({ err }, "Error creating branch");
-		res.status(err.status).json(errorMessages[err?.status || 500]);
+
+		if (err.status === 403) {
+			const user = await gitHubUserClient.getUser();
+			const gitHubConfigurationLink = await getGitHubConfigurationLink(user.data.login, jiraHost, gitHubAppConfig.hostname, owner, repo);
+			res.status(err.status).json({ error: getErrorMessages(err?.status, gitHubConfigurationLink) });
+		} else {
+			res.status(err.status || 500).json({ error: getErrorMessages(err?.status || 500) });
```

Comment:
no need in `?` here too, because it would be undefined and that's OK
Also, since we have code duplication, I'd suggest to have a temp variable
`const status = err.status || 500`

Reply:
Removed the `OR` part, cause status is always present.